salt/engines/slack.py

```diff
@@ -88,6 +88,13 @@
 
 :depends: slackclient
 
+
+.. note:: groups_pillar_name
+
+    In order to use this, the engine must be running as of a minion running on
```

Comment:
What does `as of a minion` mean here?

Reply:
put your `engines:` settings in /etc/salt/minion, and start a salt-minion process with them.